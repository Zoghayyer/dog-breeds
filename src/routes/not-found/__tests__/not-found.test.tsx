import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../../not-found';

describe('(Component) NotFound', () => {
  let component;

  const shallowNotFound = (extendedProps = {}) => {
    component = shallow(<NotFound  {...extendedProps} />);
  };

  beforeEach(() => shallowNotFound());

  it('renders not found component', () => {
    expect(component).toMatchSnapshot();
  });
});
