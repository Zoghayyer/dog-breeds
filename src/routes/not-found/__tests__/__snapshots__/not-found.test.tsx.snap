// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) NotFound renders not found component 1`] = `
<div
  className="not-found"
>
  <div
    className="mb-4"
  >
    <span
      className="not-found-message"
    >
      Sorry! We couldn't find the page you were looking for.
    </span>
  </div>
  <div>
    <Link
      className="navigation-link"
      to="/"
    >
      <span>
        Go to the main page
      </span>
    </Link>
  </div>
</div>
`;
